client/components/notice/index.tsx

```diff
@@ -0,0 +1,146 @@
+/**
+ * External dependencies
+ */
+
+import React, { useEffect, FunctionComponent, useRef } from 'react';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Internal dependencies
+ */
+import { TimerHandle } from 'calypso/types';
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Module constants
+ */
+const GRIDICONS_WITH_DROP = [
+	'add',
+	'cross-circle',
+	'ellipsis-circle',
+	'help',
+	'info',
+	'notice',
+	'pause',
+	'play',
+	'spam',
+];
+
+/**
+ * Constants
+ */
+enum Status {
+	Error = 'is-error',
```

Comment:
Just like above, should we sort these alphabetically?

Reply:
Good idea, done!